### PR TITLE
Notification refactoring

### LINKED ISSUE #453: visual feedback missing when changing contact notifications
Changing the notification sound and vibrate options for a contact have no visual effect in the settings, they are set in the background but not visually in the contact is settings, in the case of the vibrate option if you go back an open the settings again it is updated with the user selection but the sound option still don't show the selected value.

- Platform (android/blackberry/anbox): Android 6.0
- Device: BLU
- Delta Chat Version: 0.92
- Steps to reproduce the problem: Go to a contact is profile and try to change the the notification sound  and vibrate options, they are changed internally but not visually.

### PR DESCRIPTION
**Pull request by cyBerta**
* tested with android 4.1, 7.0 and 8.0
* show images in notifications
* reduce the noise on Android 0+
* fixed the summary notification layout for pre API 23: latest chats above
* removed dead signal code
* fixed  #453 (missing visual feedback for last selected notification sound)

### CODE REVIEW

**Comment by cyBerta** [comment]
@r10s I think I've finally made it... ready for review

File: src/org/thoughtcrime/securesms/ProfileActivity.java [hunk 1]
```diff
@@ -385,13 +385,16 @@ public void onSoundSettings() {
   }
 
   public void onVibrateSettings() {
+    int checkedItem = Prefs.getChatVibrate(this, chatId).getId();
+    int[] selectedChoice = new int[]{checkedItem};
     new AlertDialog.Builder(this)
```
Comment (ralphtheninja) [inline src/org/thoughtcrime/securesms/ProfileActivity.java]: Can we change this? It should be a function instead.

Comment (ralphtheninja) [reply src/org/thoughtcrime/securesms/ProfileActivity.java]: Nevermind, I thought this was JavaScript ;)

Comment (ralphtheninja) [reply src/org/thoughtcrime/securesms/ProfileActivity.java]: Just .. ignore me. It's for the best :wink:

Comment (r10s) [reply src/org/thoughtcrime/securesms/ProfileActivity.java]: :)

**Comment by r10s** [comment]
lgtm, merging. i think if there are still problems, they should be targeted in a new pr.

**Pull request merged**